Media/DisplayMedia/Strategies/ImageStrategy.php

```diff
@@ -65,7 +65,13 @@ public function displayMediaForWysiwyg(MediaInterface $media, $format = '')
      */
     public function getMediaFormatUrl(MediaInterface $media, $format)
     {
-        return $this->getFileUrl($media->getFilesystemName(), $format);
+        $key = $media->getFilesystemName();
```

Comment:
@SOLenG : in this project we avoid as possible the 'else' statments. You can ask @nicolasThal for info about that.